composer.json

```diff
@@ -36,7 +36,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php-64bit": ">=7.2",
```

Comment:
Didnt know this exists

Reply:
Neither did I until working on this ticket, but https://getcomposer.org/doc/01-basic-usage.md#platform-packages makes it sound like it'll work how we'd want.